batch-models/src/main/scala/org/ekstep/analytics/model/DruidQueryProcessingModel.scala

```diff
@@ -46,7 +48,19 @@ object DruidQueryProcessingModel extends IBatchModelTemplate[DruidOutput, DruidO
 
   override def preProcess(data: RDD[DruidOutput], config: Map[String, AnyRef])(implicit sc: SparkContext, fc: FrameworkContext): RDD[DruidOutput] = {
 
-    setStorageConf(getStringProperty(config, "store", "local"), config.get("accountKey").asInstanceOf[Option[String]], config.get("accountSecret").asInstanceOf[Option[String]])
+    val storeType = config("reportConfig").asInstanceOf[Map[String, AnyRef]].getOrElse("storeType", None)
+    var accountKey = config.get("accountKey").asInstanceOf[String]
+    var accountSecret = config.get("accountSecret").asInstanceOf[String]
+
+    if (storeType == "private") {
+      accountKey = "azure_storage_key"
+      accountSecret = "azure_storage_secret"
+    } else if (storeType == "public") {
+      accountKey = "reports_storage_key"
+      accountSecret = "reports_storage_secret"
+    }
```

Comment:
1.Don't use var for accountKey & accountSecret also take key & secret from config instead of hardcoding.
2. `storeType == "private"` - please use equals method instead